plugins/experimental/tls_bridge/CMakeLists.txt

```diff
@@ -17,5 +17,5 @@
 
 add_atsplugin(tls_bridge tls_bridge.cc)
 
-target_link_libraries(tls_bridge PRIVATE ts::tsutil libswoc::libswoc)
```

Comment:
Isn't the underlying issue that `tls_bridge` depends on this? This will fix the error, but it won't fix the problem.

Reply:
It uses `tscore/Regex.h`; maybe it could use PCRE directly as the slice plugin does.